ledfx/config.py

```diff
@@ -588,6 +588,13 @@ def sanitise_effect_config(effect_type, old_config):
                 "FXMatrix devices are no longer supported. Add it as plain UDP or WLED."
             )
             continue
+        if (
+            device["type"].lower() == "artnet"
+            and "device_repeat" in device["config"]
+        ):
+            device["config"]["pixels_per_device"] = device["config"][
+                "device_repeat"
+            ]
```

Comment:
We should probably pop off the device["config"]["device_repeat"] at this point, or it will stick around forever and trigger this clause in the future, replacing any new value of "pixels_per+device" with the legacy value of device_repeat

device["config"].pop("device_repeat", None)

Reply:
Confirmed that the device_repeat value does indeed stick around, otherwise all works. So get that fixed and we can check this in...